src/features/MovieCategories/component.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import './style.scss';
+
+export default class MovieCategories extends Component 
+{
+    // define our state
+    constructor(props)
+{
+        super(props);
+    }
+
+    render() 
+{
+        return (
+	<aside className='movieSection-categories'>
+		<ul className='category-items'>
+			{this.props.categories.map((item, index) => (
```

Comment:
since callback of the map is and anonymous function, it'll be recreated on every render for no reason
I'd recommend moving it in component methods instead